docs/prometheus-exporter.md

```diff
@@ -116,3 +106,20 @@ The block height of the last arrived block.
 Timestamp for the processed last arrived block (as Unix time in milliseconds).
 
 Note that this is the time when the node has last processed a new block. It is **not** the objective timestamp of the block (i.e., slot time).
+
+### `consensus_received_messages_total`
+
+Total number of consensus messages received. Labelled with message type (`message=<type>`) and the outcome (`result=<outcome>`).
+
+Possible values of `message` are:
+- `"block"`
+- `"transaction"`
+- `"finalization record"`
+- `"finalization message"`
+- `"catch-up status message"`
+
+Possible values of `result` are:
+- `"valid"` Successful outcome.
+- `"invalid"` Messages being rejected as invalid.
+- `"dropped"` Messages being dropped due to a full queue. See either `network_inbound_low_priority_message_queue_size` or `network_inbound_high_priority_message_queue_size` for more on this.
+- `"duplicate"` Duplicate consensus messages (this is not including deduplication of the network layer).
```

Comment:
Please clarify what is meant by "this is not including ..." part. It is not going to be clear to readers. Explain what the value is measuring instead.

Reply:
I have used something similar to what you suggested in the previous comments